python/plugins/db_manager/db_plugins/spatialite/connector.py

```diff
@@ -571,6 +571,14 @@ def runVacuum(self):
         c.execute('VACUUM')
         self.connection.isolation_level = '' # reset to default isolation
 
+    def commentTable(self, schema, tablename, comment=None):
+        """Comment the table"""
+        return ''
+
+    def getComment(self, tablename, field):
```

Comment:
Move this method `getComment` into the base class `DBConnector` as a default implementation and override in PG connector only, instead of providing the same default implementation in all connectors but the PG one.